core/src/main/scala/org/apache/spark/deploy/ExternalShuffleService.scala

```diff
@@ -58,9 +62,42 @@ class ExternalShuffleService(sparkConf: SparkConf, securityManager: SecurityMana
 
   private val shuffleServiceSource = new ExternalShuffleServiceSource
 
+  protected def initRegisteredExecutorsDB(dbName: String): File = {
```

Comment:
I think renaming to `findRegisteredExecutorsDBFile` would be better, nothing is really getting initialized here

Reply:
ok,done